test/Microsoft.ML.Tests/Scenarios/TensorflowTests.cs

```diff
@@ -22,123 +25,47 @@ public void TensorFlowTransformCifarLearningPipelineTest()
-            pipeline.Add(new StochasticDualCoordinateAscentClassifier());
-
-            var model = pipeline.Train<CifarData, CifarPrediction>();
-            string[] scoreLabels;
-            model.TryGetScoreLabelNames(out scoreLabels);
-
-            Assert.NotNull(scoreLabels);
-            Assert.Equal(3, scoreLabels.Length);
-            Assert.Equal("banana", scoreLabels[0]);
-            Assert.Equal("hotdog", scoreLabels[1]);
-            Assert.Equal("tomato", scoreLabels[2]);
-
-            CifarPrediction prediction = model.Predict(new CifarData()
+                Column = new[]
+                    {
+                        new TextLoader.Column("ImagePath", DataKind.TX, 0),
+                        new TextLoader.Column("Label", DataKind.TX, 1),
+                    }
+            }, new MultiFileSource(dataFile));
+
+            var pipeEstimator = new ImageLoadingEstimator(env, imageFolder, ("ImagePath", "ImageReal"))
+                    .Append(new ImageResizingEstimator(env, "ImageReal", "ImageCropped", imageHeight, imageWidth))
+                    .Append(new ImagePixelExtractingEstimator(env, "ImageCropped", "Input", interleave: true))
+                    .Append(new TensorFlowEstimator(env, model_location, new[] { "Input" }, new[] { "Output" }))
+                    .Append(new ColumnConcatenatingEstimator(env, "Features", "Output"))
+                    .Append(new ValueToKeyMappingEstimator(env, "Label"))
+                    .Append(new SdcaMultiClassTrainer(env));
+
+
+            var transformer = pipeEstimator.Fit(data);
+            var predictions = transformer.Transform(data);
+
+            var metrics = env.MulticlassClassification.Evaluate(transformer.Transform(data));
+            Assert.Equal(1, metrics.AccuracyMicro, 2);
+
+            var predictFunction = transformer.MakePredictionFunction<CifarData, CifarPrediction>(env);
+            var prediction = predictFunction.Predict(new CifarData()
             {
                 ImagePath = GetDataPath("images/banana.jpg")
             });
-            Assert.Equal(1, prediction.PredictedLabels[0], 2);
-            Assert.Equal(0, prediction.PredictedLabels[1], 2);
-            Assert.Equal(0, prediction.PredictedLabels[2], 2);
+            Assert.Equal(1, prediction.PredictedScores[0], 2);
+            Assert.Equal(0, prediction.PredictedScores[1], 2);
+            Assert.Equal(0, prediction.PredictedScores[2], 2);
 
-            prediction = model.Predict(new CifarData()
+            prediction = predictFunction.Predict(new CifarData()
             {
                 ImagePath = GetDataPath("images/hotdog.jpg")
             });
-            Assert.Equal(0, prediction.PredictedLabels[0], 2);
-            Assert.Equal(1, prediction.PredictedLabels[1], 2);
-            Assert.Equal(0, prediction.PredictedLabels[2], 2);
-        }
-
-        [Fact(Skip = "Model files are not available yet")]
-        public void TensorFlowTransformInceptionPipelineTest()
```

Comment:
>TensorFlowTransformInceptionPipelineTest [](start = 20, length = 40)

Looks like this test is not doing anything / gets skipped. Model files not available  for this test  + it uses Legacy API.    Deleting.